run.sh

```diff
@@ -6,4 +6,18 @@ fi
 # make rabbit own its own files
 chown -R rabbitmq:rabbitmq /var/lib/rabbitmq
 
-exec /usr/sbin/rabbitmq-server
+if [ -z "$CLUSTER_WITH" ] ; then
+    /usr/sbin/rabbitmq-server
+else
+    if [ -f /.CLUSTERED ] ; then
+        /usr/sbin/rabbitmq-server
+    else
+        touch /.CLUSTERED
+        /usr/sbin/rabbitmq-server -detached
+        rabbitmqctl stop_app
+        rabbitmqctl join_cluster rabbit@$CLUSTER_WITH
+        rabbitmqctl start_app
+        tail -f /var/log/rabbitmq/rabbit\@$HOSTNAME.log
```

Comment:
Thank you, I modified tail option.

As explained in the clustering documentation : https://www.rabbitmq.com/clustering.html I don't think there is an other way to manage clustering, showing the log file is what doing /usr/sbin/rabbitmq-server. 

What are you thinking about with supervisord ? I will have to run a command in the same way.